server/utils.js

```diff
@@ -101,3 +103,17 @@ exports.stringTemplate = (configPath, ...args) => {
 
   return ''
 }
+
+const readFileAsync = promisify(fs.readFile)
```

Comment:
I went with `promisify` because I don't think the `fs.promises` API is supported in Node 8.

Reply:
Ah, that's a good catch. Let's reassess at Node 8 EOL in several months, but for now this seems like a good way to go.